RemoteInput/Plugin/Python/PythonShared.cxx

```diff
@@ -0,0 +1,1087 @@
+#include "PythonShared.hxx"
+#include "NativePlugin.hxx"
+
+// MARK: Non-EIOS
+
+PyObject* PythonCreateClient(PyObject* self, PyObject* args[], Py_ssize_t args_length) noexcept
+{
+    std::int32_t pid = from_python_object<std::int32_t>(args[0]);
```

Comment:
What happens if I wanted to create a client by name? Such as: `PythonCreateClient("ProcessName.exe")`
We should have a function that can do that as well, since we have `Inject_PID` and `Inject("process_name")`

Reply:
I think this function should only be PID, since thats the flow it uses, but I will add the inject by name function.